api-reference/docstubs/api/user-reminderview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "reminderView"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# reminderView
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /me/reminderView
+GET /users/{usersId}/reminderView
+GET /me/managedDevices/{managedDeviceId}/users/{userId}/reminderView
+```
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}.Required|
+
```

Comment:
This comment is just for the section generation logic - If the generation code assumed this API takes on a request body, then should it also include the Content-Type request header?
(Even though in reality per my next comment, this function does not take on a request body, in which case the Content-Type header is not needed.)

Reply:
Also a question about generating request headers - do we have a plan to have partners include supported headers in the CSDL at some point in the future?
This function btw supports a Prefer time zone request header. Just wondering if this would be automated or partners will manually add to the generated topic.

Reply:
I think this is manual for now...